sass/layout/_legacy-sidebar-left.scss

```diff
@@ -177,10 +177,13 @@
         position: absolute;
         margin-left: 50%;
         width: 188px;
+        height: 32px;
```

Comment:
add `padding: 8px;` to give the button enough padding when translation is longer